### ISSUE TITLE
Feature request: Allow for trailing newlines in include files

### ISSUE THREAD

**Issue by cncook001** [mondeja/mkdocs-include-markdown-plugin#98]
I have a file called "test.md"

```
1.  This is the first number line
1.  {%
       include-markdown "./includes/content_chunk2.md"
    %}

1.  If everything works as expected this should be number 3
```

The contents of "./includes/content_chunk2.md" is this

```
This is my second content chunk
```

The challenge is using prettier. It will automatically ensure there is a final "\n" at the end of the file. This breaks the number scheme rendering.

The rendered file will be like this:

```
Test

1.    This is the first number line
2.    This is my second content chunk

1.    If everything works as expected this should be number 3
```

The trailing newline can be stripped with this command:

```
for i in docs/includes/*.md; do printf "$( <$i )" > $i; done
```

Then it renders like this:
```
Test

1.    This is the first number line
2.    This is my second content chunk
3.    If everything works as expected this should be number 3
```

Is there a way to allow an include file to have a trailing newline and also render the numbering system?

**Comment by mondeja** [comment]
Interesting. I think that we can add an option named `trailing-newlines` by default as `true` that when would be `false` would remove the trailing newlines if the file to include have them 👍🏼

**Comment by mondeja** [comment]
As a workaround, you can currently do someting like this:

```markdown
1. List item number 1
1. {%
       include-markdown "{filepath}"
       comments=false
    %}1. List item number 3
```

**Comment by mondeja** [comment]
Thanks for the report. This feature has been released in [v3.5.0](https://github.com/mondeja/mkdocs-include-markdown-plugin/releases/tag/v3.5.0).
